skimage/registration/tests/test_tvl1.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from skimage._shared import testing
+from skimage.registration import optical_flow_tvl1
+
+
+def test_no_motion_2d():
+    rnd = np.random.RandomState(0)
+    img = rnd.normal(size=(256, 256))
+
+    flow = optical_flow_tvl1(img, img)
+
+    assert np.all(flow == 0)
+
+
+def test_no_motion_3d():
```

Comment:
Great! This will at least exercise the code. =)

Another test would be to use ndimage transforms that are expected to produce a particular pattern of flow. For example, wouldn't shrinking the image about its center produce a radially shrinking field with predictable flow? Or, more simply, a translation of x, y, z pixels should produce that flow uniformly across the image, right?

Reply:
Yes, good suggestion, but the resulting optical flow will not be equal to the ground truth, all we can do is expect a result reasonably close to the true solution. I will give this a try ASAP ;-)